packages/shared/lib/core/account/interfaces/account-balances.interface.ts

```diff
@@ -0,0 +1,10 @@
+export interface IAccountBalances {
+    aliases: unknown[]
+    available: number
+    foundries: unknown[]
+    nativeToken: Record<string, unknown>
+    nfts: unknown[]
+    potentiallLockedOutputs: unknown[]
+    requiredStorageDeposit: number
+    total: number
+}
```

Comment:
This one you can use from wallet.rs. It's called `AccountBalance` and can be found here: `wallet.rs/bindings/nodejs/types/account.ts`

Reply:
Ahh yes didn't exist at the time

Reply:
I think because we use this locally and not just in the api / bindings its best to type this ourselves?

Reply:
Yeah, but it might be subject to changes, so if we type it ourselves, it will be more difficult to keep track of it.

Reply:
Okay will update in the current Pr